ui/app/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -49,22 +63,24 @@ export default class ConfirmApproveContent extends Component {
           'confirm-approve-content__card--no-border': noBorder,
         })}
       >
-        <div className="confirm-approve-content__card-header">
-          <div className="confirm-approve-content__card-header__symbol">
-            {symbol}
-          </div>
-          <div className="confirm-approve-content__card-header__title">
-            {title}
-          </div>
-          {showEdit && (
-            <div
-              className="confirm-approve-content__small-blue-text cursor-pointer"
-              onClick={() => onEditClick()}
-            >
-              Edit
+        {showHeader && (
+          <div className="confirm-approve-content__card-header">
+            <div className="confirm-approve-content__card-header__symbol">
+              {symbol}
             </div>
-          )}
-        </div>
+            <div className="confirm-approve-content__card-header__title">
+              {title}
+            </div>
+            {showEdit && (
+              <div
```

Comment:
This one should also be a button, the cursor-pointer class is deprecated as well so we shouldn't use that going forward (this isn't documented well, no way you could have really known). If we make it a button though we will get some accessibility for free and buttons are meant to handle clicks.